src/webview/mod.rs

```diff
@@ -643,6 +657,7 @@ pub trait WebViewBuilderExtWindows {
   ///
   /// By default wry passes `--disable-features=msWebOOUI,msPdfOOUI,msSmartScreenProtection`
   /// so if you use this method, you also need to disable these components by yourself if you want.
+  /// Furthermore you need to add `--autoplay-policy=no-user-gesture-required` if autoplay is enabled.
```

Comment:
Does this still apply? I saw there's a if statement in webview2's code.

Reply:
Yes, the additional_browser_args settings will overwrite the autoplay flag similar to how it overwrites wry's default --disable-features stuff.

But i think i messed up the wording a bit, i'll try to make it more clear.